schedule/yam/agama_rmt_unattended_s390x.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: agama_rmt_s390x
+description: >
+  Perform agama installation with rmt registration
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama
+  - boot/reconnect_mgmt_console
+  - installation/first_boot
+  - console/validate_repos
```

Comment:
this would need a test data file, maybe you can guess the content even with the bug fix on-going.